book/src/feature/handler.md

```diff
@@ -45,3 +45,19 @@ impl MyExecutor {
 }
 ```
 
+### Example: Use a simple handler
+
+```rust,noplayground
+// api.rs
+
+use flutter_rust_bridge::handler::ReportDartErrorHandler as RDEH;
+use flutter_rust_bridge::handler::SimpleHandler as SH;
+use flutter_rust_bridge::handler::ThreadPoolExecutor as TPE;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref FLUTTER_RUST_BRIDGE_HANDLER: SH<TPE<RDEH>, RDEH> = SH::new(
+        TPE::new(RDEH), RDEH {}
+    );
```

Comment:
uber-nit: personally prefer not using abbr, especially in the docs, where readability may be quite useful

Reply:
Okay I'll fix it right away